app/src/main/java/org/apache/fineract/utils/FileUtils.java

```diff
@@ -224,7 +248,7 @@ public static void writeInputStreamDataToFile(InputStream in, File file) {
             out.close();
             in.close();
         } catch (Exception e) {
-            Log.d(LOG_TAG, e.getLocalizedMessage());
+            Logger.getLogger(e.getLocalizedMessage());
```

Comment:
All other looks good to me but why did you replace Log with logger , it isn't a essential change . So i'd suggest you keep t earlier one only so that it will be easy and straight forward to merge this PR.

Reply:
Also , can you please tell me the steps to navigate to the edit profile image page?

Reply:
It you would run check on your machine you would have experienced pmd fail due to log messages. I prefer in simplicity too but it is necessity. Please try check with log message you would find pmd error

Reply:
If you open customer and click image above there would be edit icon above there. These are stepsto edit image of customer

Reply:
you didn't get me . What I am trying to say is don't fix 2 issues in a single PR like you are getting the issue of pmd error with LOG, then update this in a diff. PR or ask this on the IRC channel . The use of Logger might be beneficial , just don't update two things at the same time. Also , when some newbie goes thorough your PR, he won't understand why is this Log been replaced with Logger in this particular PR. This modification isn't related to this issue, right ? I hope that makes sense :)

Reply:
Ok i got what you said but if i made changes in different pr it would be for a small change, also if i dont make this chages here my build will fail. Thus it is necessary to make changes here.

Reply:
Ohh Okay then!